crates/sqlexec/src/remote/planner.rs

```diff
@@ -185,12 +192,67 @@ impl ExtensionPlanner for DDLExtensionPlanner {
                 })))
             }
             ExtensionType::DropTables => {
-                let lp = require_downcast_lp::<DropTables>(node);
-                Ok(Some(Arc::new(DropTablesExec {
-                    catalog_version: self.catalog_version,
-                    tbl_references: lp.tbl_references.clone(),
-                    if_exists: lp.if_exists,
-                })))
+                let tmp_catalog = session_state
```

Comment:
this whole section inspects the catalogs & builds either a `DropTempTablesExec` or a `DropTablesExec`